bluesky/tests/test_callbacks.py

```diff
@@ -449,3 +451,24 @@ def test_plotting_hints(RE, hw, db):
     RE(grid_scan([hw.det], hw.motor1, -1, 1, 2, hw.motor2, -1, 1, 2,
                  True, hw.motor3, -2, 0, 2, True))
     assert dc.start[-1]['hints'] == hint
+
+
+@pytest.mark.skipif(sys.version_info < (3, 6),
+                    reason="requires python3.6 or higher")
+def test_optional_matplotlib():
+    test_src = """
+import sys
+from importlib.abc import Finder
+
+
+class MyFinder(Finder):
+    def find_module(self, name, path):
+        if name.startswith('matplotlib'):
+            raise ModuleNotFoundError
+
+sys.meta_path.insert(0, MyFinder())
+
+from bluesky.callbacks import CallbackBase
+sys.exit(int('matplotlib' is sys.modules))
```

Comment:
Should be `... in sys.modules`.

This does not help with the case when matplotlib is installed, 
but there is no DISPLAY and no plotting is intended.

Reply:
yeah, should be '`in'`.  The no display case 'just works' on Matplotlib 3 and is a configuration error on the server for older Matplotlib versions.